service/storage_service.hh

```diff
@@ -873,13 +874,18 @@ private:
     future<> update_topology_with_local_metadata(raft::server&);
     void set_topology_change_kind(topology_change_kind kind);
 
-public:
     // This is called on all nodes for each new command received through raft
     // raft_group0_client::_read_apply_mutex must be held
     // Precondition: the topology mutations were already written to disk; the function only transitions the in-memory state machine.
-    // Public for `reload_raft_topology_state` REST API.
     future<> topology_transition();
 
+public:
+    // Reloads in-memory topology state safely under group 0 read_apply mutex.
+    // Does not modify on-disk state.
+    //
+    // Must be called on shard 0.
+    future<> reload_raft_topology_state(service::raft_group0_client&);
```

Comment:
missing the parameter name - not sure if actually needed here (what's the general approach) as the type name is explanatory enough, just noticed

Reply:
I always skip parameter name in such cases (if it just repeats the type name basically)